randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1188,9 +1188,14 @@ Extra - room_id: [38]
       All of the following:
           Speed Booster
           Any of the following:
-              Wave Beam or Can Use Bombs
-              # Shinespark directly from adjacent room to the item
-              Screw Attack and Varia Suit and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando and Can Use Power Bombs
+              Can Use Bombs
+              # Wave Beam is difficult to control, but can solve the puzzle (Wide Beam could be turned off if also collected)
+              Wave Beam and Knowledge (Advanced)
+              All of the following:
+                  # Shinespark directly from adjacent room to the item
+                  Screw Attack and Varia Suit and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando
+                  # Power Bombs requires Bombs at the moment, but if that changes, Power Bombs alone will not allow the puzzle to be solved in order to leave the room
```

Comment:
This is fine, the exit requirements are currently tied in the adjacent room.
This allows the possibility to have to shinespark in as a PONR for bombs/wave in the puzzle to escape, which is great :)